Scripts/gen_Digitize.sh

```diff
@@ -51,7 +51,8 @@ elif [[ "${digitype}" == "OnSpill" ]]; then
   echo outputs.TriggeredOutput.fileName: \"dig.owner.${digout}Triggered.version.sequencer.art\" >> digitize.fcl
   echo outputs.UntriggeredOutput.fileName: \"dig.owner.${digout}Untriggered.version.sequencer.art\" >> digitize.fcl
 fi
-generate_fcl --dsconf="$conf" --dsowner=mu2e --description="${digout}" --embed digitize.fcl \
+
+generate_fcl --dsconf="$conf" --dsowner=mu2e --description="${digout}Digi" --embed digitize.fcl \
```

Comment:
I don't think the --description field is used in the output fcl: they are either *OnSpillTriggered, *OffSpillUntriggered, etc.  What difference do you see in the output after making this change?

Reply:
The difference is literally in the name of the fcl files. If we don't add `Digi` we have something like `cnf.mu2e.CosmicCORSIKAExtractedNoField.MDC2020o_10h.1000.fcl` which is the same name of the fcl file for the dts stage.